tests/hurl/blocks.hurl

```diff
@@ -0,0 +1,35 @@
+POST {{url}}
+```graphql
+query Blocks($limit: Int = 10, $sort_by: BlockSortByInput!, $query: BlockQueryInput!) {
+  blocks(limit: $limit, sortBy: $sort_by, query: $query ) {
+    stateHash
+    txFees
+    blockHeight
+    canonical
+    creator
+    dateTime
+    receivedTime
+    snarkFees
```

Comment:
Doesn't the [frontend require more fields](https://github.com/Granola-Team/mina-block-explorer/blob/main/graphql/queries/blocks.graphql)?

Reply:
It does and will be handled in a future PR.